src/SelectMulti.vue

```diff
@@ -425,7 +429,7 @@
 				</div>
 			</slot>
 			</div>
-			<div class="combo-input-icon-block" :class="{ 'not-clickable': !$slots['input-icon'] }">
+			<component :is="iconIsClickable ? 'button' : 'div'" @click="iconIsClickable ? updateMenuState(iconIsClickable) : false" class="combo-input-icon-block" :class="{ 'not-clickable': !iconIsClickable }">
```

Comment:
To make it clear that this button is meant to always open 
```suggestion
			<component :is="iconIsClickable ? 'button' : 'div'" @click="iconIsClickable ? updateMenuState(true) : false" class="combo-input-icon-block" :class="{ 'not-clickable': !iconIsClickable }">
```